core/src/main/scala/org/apache/spark/scheduler/DAGScheduler.scala

```diff
@@ -1939,24 +1941,24 @@ private[spark] class DAGScheduler(
       hostToUnregisterOutputs: Option[String],
       maybeEpoch: Option[Long] = None): Unit = {
     val currentEpoch = maybeEpoch.getOrElse(mapOutputTracker.getEpoch)
+    logDebug(s"Considering removal of executor $execId; " +
```

Comment:
nit: `;` -> `,`

Reply:
I deliberately used a semicolon; it was not a typo.